### PR TITLE
#319 Added --debug flag for the up command

### PR DESCRIPTION
**Pull request by frankscholten**

### CODE REVIEW

File: minimesos/src/main/java/com/containersol/minimesos/main/CommandUp.java [hunk 1]
```diff
@@ -116,6 +125,13 @@ public void setClusterConfigPath(String clusterConfigPath) {
 
     @Override
     public void execute() {
+        if (debug != null) {
+            LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
+            Logger rootLogger = loggerContext.getLogger(Logger.ROOT_LOGGER_NAME);
+            rootLogger.setLevel(Level.DEBUG);
+            rootLogger.debug("Initialized debug logging");
```
Comment (sadovnikov) [inline minimesos/src/main/java/com/containersol/minimesos/main/CommandUp.java]: I think you meant `LOGGER.debug("Initialized debug logging");` ?

File: minimesos/src/main/resources/logback.xml [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
 
-    <root level="info">
+    <logger name="com.containersol.minimesos.main" level="INFO">
         <appender-ref ref="STDOUT" />
-    </root>
+    </logger>
 
-</configuration>
+    <root level="info">
+        <appender-ref ref="STDERR" />
```
Comment (sadovnikov) [inline minimesos/src/main/resources/logback.xml]: should `root` use both STDOUT and STDERR, while STDERR logs only WARN, ERROR and FATAL?

**Comment by sadovnikov** [comment]
approved

**Comment by sadovnikov** [comment]
Approved

**Comment by alex-glv** [comment]
I can't see anything wrong with the pull request.
However, I'd suggest making --debug flag accept logging levels, (debug, info, error, fatal?)

**Comment by alex-glv** [comment]
LGTM

**Comment by sadovnikov** [comment]
Approved

**Pull request merged**